views/_includes/input-text.pug

```diff
@@ -0,0 +1,31 @@
+//- `label`: text for the label
+//- `divClasses`: any classes to be applied to the wrapper div (but not the input itself)
+//- `hintText`: any text that appears above the input in light grey (ex. Optional)
+
+mixin textInput(label, divClasses, hintText)
+  div(
+    class={['has-error']: errors && errors[attributes.name]}
+    class=divClasses
+  )
+    label(
+      for!=attributes.name 
+      id!=attributes.name + '__label'
+    ) #{__(label)}
+    if hintText
+      span.cra-form-message #{__(hintText)}
+    if errors && errors[attributes.name]
+      +validationMessage(
+        errors[attributes.name].msg, 
+        attributes.name
+      )
+    if attributes.class.includes('input-with-unit')
+      span.input-with-unit__unit(
+        id!=attributes.name+'__unit', 
+        for!=attributes.name
```

Comment:
we can remove this `for`, since that was actually something I left in by accident. 

I used to have this span as a `<label>` before (so I added a `for` attr) because I saw it here: http://alphagov.github.io/digitalmarketplace-frontend-toolkit/forms/textbox.html#question-100

- adding a label: https://github.com/cds-snc/cra-claim-tax-benefits/commit/1fadc0e708ca3a8fef280bdf16117fa67f9e159a
- changing to a span: https://github.com/cds-snc/cra-claim-tax-benefits/commit/2ebfef80d3ab0d8a1860f5dd3a898d7a6ab47ca2

Reply:
ahhh okay. Removed it!